### DISCUSSION TITLE
Align notifications right-most after date/time hidden

### DISCUSSION THREAD

**Discussion by luisgcr** [jmalarcon/Win11ClockToggler#11] [Ideas]
I'm using W11 and changed the registry adding the new Key/Value in order to reduce the size of taskbar icons

Path: Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced
Key: TaskbarSi (DWORD32)
Value: 0 (0=smal, 1=default, 2=large)

After that the Date and Time does not fit and the date is becoming truncated , so I tried your solution Win11ClockToggler but when I hide the notifications are not aligned to the most-right position
Is there a way to realign the notifications to the right after hiding the date/time ?

Or at least show the Time/Date in just one line instead of two-lines (that was truncated) ?
Maybe add this feature on the code.

**Comment by jmalarcon** [comment]
Hi:

Sorry, AFAIK there's no way in Windows 11 to align the notification area to the right when the clock is hidden. In Windows 10 I found a hack to achieve it, but Win11 is a totally different beast and I couldn't find a way to do it.
